lib/customers.js

```diff
@@ -281,6 +281,18 @@ function enhanceAtFields (fields) {
   return _.merge(fields, atFields)
 }
 
+function updateRawData (fields) {
+  const fieldsWithRawData = [
+    'id_card_data'
+  ]
+
+  const fieldsToUpdate = _.pick(fieldsWithRawData, fields)
+  const rawFields = _.fromPairs(_.map(f => [`${f}_raw`, JSON.stringify(fieldsToUpdate[f].raw)], _.keys(fieldsToUpdate)))
+  const fieldsWithoutRawData = _.mapValues(f => _.omit(['raw'], f), fieldsToUpdate)
+
+  return _.merge(fieldsWithoutRawData, rawFields)
+}
+
```

Comment:
isn't this removing any other fields and just updating id_card_data?

Reply:
You are correct! This next commit should solve this, nice catch